soknad/src/main/java/no/nav/k9/søknad/ytelse/omsorgspenger/v1/OmsorgspengerUtbetalingValidator.java

```diff
@@ -74,10 +74,14 @@ List<Feil> validerMedGyldigEndringsperodeHvisDenFinnes(Ytelse ytelse, List<Perio
     }
 
     List<Feil> validerMotGyldigePerioder(OmsorgspengerUtbetaling omp, List<Periode> gyldigeEndringsperioder) {
-        final LocalDateTimeline<Boolean> søknadsperiodeTidslinje = lagTidslinjeOgValider(List.of(omp.getSøknadsperiode()), "søknadsperiode.perioder");
+        if (omp.getFraværsperioderKorrigeringIm() != null && !omp.getFraværsperioderKorrigeringIm().isEmpty()) {
```

Comment:
Validering mot gyldige perioder skal ikke skje for korrigering IM?

Reply:
Jeg tror ikke det. Skal det?

Reply:
Jeg har nå uansett lagt til støtte for det :)